### ISSUE TITLE
Long running process

### ISSUE THREAD

**Issue by CesarGallego** [BBVA/kapow#86]
Some process are heavy, so you need keep them alive. We need this kind of endpoint?

**Comment by dumblob** [comment]
Yes, something like "streaming" from such long running processes seems necessary.

Especially nowadays when HTTP3 (built upon QUIC) became a streaming technology instead of request-response as older HTTP versions.
